---
torpedo_task:
  set_torpedo_depth:
    sleep_duration: 3.0
  find_and_aim_torpedo:
    full_rotation: false
    set_frame_duration: 4.0
    rotation_speed: 0.3
  wait_for_aligning_start:
    delay_time: 4.0
  set_torpedo_close_approach_frame:
    radius: 0.4
  wait_for_close_approach_complete:
    delay_time: 7.0
  launch_torpedo_1:
    id: 1
  wait_for_torpedo_launch:
    delay_time: 6.0
  launch_torpedo_2:
    id: 2
  wait_for_torpedo_2_launch:
    delay_time: 3.0
  set_torpedo_exit_depth:
    depth: -0.7
    sleep_duration: 3.0
